apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -68,7 +68,10 @@ run(Queue, F) ->
         error:{rejected, _} ->
             {503, [], #{reason => <<"Temporary overload">>}};
         error:timeout ->
-            {503, [], #{reason => <<"Not yet started">>}}
+            {503, [], #{reason => <<"Not yet started">>}};
+        Class:Reason:Stacktrace ->
```

Comment:
If we don't catch the crash - it propagates all the way to `cowboy_rest` where it is caught. In this case a standard code 500 response is being returned, sadly it does not have CORS headers. So we handle the error and the response follows the expected path (and has CORS headers added to it).